### PR TITLE
Tax20percent

### PR DESCRIPTION
**Pull request by OthmanSO**
- create project for the business logic and testing 
- create the priceBeforeTax functionality 
- create the priceAfterTax functionality

### CODE REVIEW

File: Price_Calculator_Testing/UnitTest1.cs [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+namespace Price_Calculator_Testing
+{
+    public class UnitTest1
+    {
+        [Fact]
+        public void PriceBeforTax_InputProduct_ReturnDoublePrice()
+        {
+            PriceCalculator pc = new PriceCalculator();
+            var prod = GenerateProduct("The Little Prince", 12345, 20.25);
```
Comment (RandaZraik) [inline Price_Calculator_Testing/UnitTest1.cs]: move these numbers to InlineData, so you can test against multiple inputs.

File: KATA/PriceCalculator.cs [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace KATA
+{
+    public class PriceCalculator
+    {
+        public static void Main()
```
Comment (RandaZraik) [inline KATA/PriceCalculator.cs]: lets move this out to Program.cs. So we make your calculator cleaner.

File: KATA/PriceCalculator.cs [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+        {
+            PriceCalculator priceCalculator = new PriceCalculator();
+        }
+
+        public double PriceBeforeTax(Product p) => p.price;
```
Comment (RandaZraik) [inline KATA/PriceCalculator.cs:11]: interesting :) it just returns the actual price?

Comment (OthmanSO) [reply KATA/PriceCalculator.cs:11]: imagine !

File: Price_Calculator_Testing/UnitTest1.cs [hunk 4]
```diff
@@ -0,0 +1,36 @@
+using KATA;
+using Xunit;
+using FluentAssertions;
+
+namespace Price_Calculator_Testing
+{
+    public class UnitTest1
+    {
+        [Theory]
+        [InlineData("The Little Prince", 12345, 20.25)]
```
Comment (RandaZraik) [inline Price_Calculator_Testing/UnitTest1.cs:10]: tb5lsh, add plenty :3

**Pull request merged**